tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <assert.h>
+#include <chrono>
+#include <concepts>
+#include <format>
+#include <iostream>
+#include <stdio.h>
+#include <string_view>
+#include <type_traits>
+#include <utility>
+
+using namespace std;
+using namespace chrono;
+
+#if defined(__cpp_lib_concepts) && !defined(__clang__) // TRANSITION, LLVM-48606

```

Comment:
Would it be possible to get Clang coverage for everything except for the `static_assert`s that are using `constexpr string`, or is the entire test affected by this Clang bug?

Reply:
I've added a card to the Chrono Project's To Do column about both Clang test coverage and using `_CONSTEXPR20_CONTAINER`.